src/configuration/configuration.rb

```diff
@@ -90,6 +90,8 @@ module Configuration
   # These are the same colors, but with saturation lower, and a little bit shuffled, to prevent collisions
   UNINTENSE_COLORS = %w[#dcdcdc #696969 #202020 #808080 #c0c0c0].freeze
 
+  DARK_COLOR = "#999999"
```

Comment:
```suggestion
  DARK_COLOR = '#999999'.freeze
```
if we want to listen to rubicon ;-)